crates/wasmtime/src/module.rs

```diff
@@ -1042,6 +1042,47 @@ impl Module {
         self.inner.memory_images()?;
         Ok(())
     }
+
+    // XXX: Only for use within this repo's workspace.
```

Comment:
In practice I've found that comments like this tend to not hold any real weight in practice, especially for "juicy" functionality like this that can't be replicated anywhere else. Given that the high-level functionality exposed here is always going to be supported in Wasmtime in one way or another (e.g. via backtraces) I think it's ok to make this an "official" API with some brief docs. I don't think it's necessarily important to spend a ton of time figuring out the perfect API here either, as that's why we bump major versions still.